pydefaultwordlist.ts

```diff
@@ -0,0 +1,105 @@
+export const default_keywords = [
+    "and",
+    "del",
+    "from",
+    "not", 
+    "while",
+    "as",
+    "elif",
+    "global",
+    "or",
+    "with",
+    "assert",
+    "else",
+    "if",
+    "pass",
+    "yield",
+    "break",
+    "except",
+    "import",
+    "print",
+    "class",
+    "exec",
+    "in",
+    "raise",
+    "continue",
+    "finally",
+    "is",
+    "return",
+    "def",
+    "for",
+    "lambda",
+    "try"
+]
+
+export const default_functions = [ 
+    "abs()",
+    "delattr()",
+    "hash()",
+    "memoryview()",
+    "set()",
+    "all()",
+    "dict()",
+    "help()",
+    "min()",
+    "setattr()",
+    "any()",
+    "dir()",
+    "hex()",
+    "next()",
+    "slice()",
+    "ascii()",
+    "divmod()",
+    "id()",
+    "object()",
+    "sorted()",
+    "bin()",
+    "enumerate()",
+    "input()",
+    "oct()",
+    "staticmethod()",
+    "bool()",
+    "eval()",
+    "int()",
```

Comment:
Can we remove the type casting functions like int() and bool() here? For our users I think most of the time they type `int` rather than `int()` so I think it would be better if it does not autocomplete with brackets every time

Reply:
Oops looks like now autocompleting does not autocomplete types (int, bool, str) any more. Can we add these 3 keywords to the `default_keywords` list? Other than this everything else looks good!